README.md

```diff
@@ -50,25 +51,31 @@ daily:20.10                 devel,groovy      20200721         Ubuntu 20.10
 ```
 
 ## Launch a fresh instance of the current Ubuntu LTS
+
 ```
 $ multipass launch ubuntu
```

Comment:
`ubuntu` isn't a valid alias. Maybe use `lts`? Then, we also need to update the outputs of `find`, `launch`, `info`, `shell`, etc to match the current lts version.

Reply:
This is legacy that is not related to this PR, but I agree, we should fix that as well by changing `ubuntu` to `lts`.

Reply:
Thanks for catching this!

Reply:
Done.